frontend/src/App.vue

```diff
@@ -29,6 +29,11 @@
         dev: import.meta.env.DEV,
       }
     },
+    computed: {
```

Comment:
Would isAuthorized be a better name?

Reply:
Probably more accurate, renamed to isAuthorized